dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -80,10 +86,25 @@ async def get_actor_info(self):
                 "current_raylet_id": actor_table_entry.address.raylet_id.hex(),
                 "ip_address": actor_table_entry.address.ip_address,
                 "port": actor_table_entry.address.port,
+                "ray_replica_id": "TODO",
+                "metadata": {
+                    "deployment_id": "TODO"
+                }
```

Comment:
is it possible that we leave the data out here and instead just include it in the Serve endpoint (e.g., a dictionary of actorIDs to serve replicaIDs or something? that would be preferrable

Reply:
Having this here would be easier for ingestion logic imo.
But I see the benefit of having in the serve endpoint for isolation of business logic. 

The main disadvantage for ingestion is that I have to reorganize the actors data and do the association myself. This may not be that hard but it will make the code more complex to understand and maintain on the product side.

Reply:
Based on what we decided in our discussion in standup, I'll remove this. In a future PR I'll publish this information in the Serve cluster snapshot.